src/Fasodev/Utils/Helpers.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+
+namespace Fasodev\Utils;
+
+/**
+ * Class Helpers
+ *
+ * @package Fasodev\Utils
+ */
+class Helpers
+{
+    /**
+     * Check if the environment is in production.
+     *
+     * @return bool
+     */
+    public static function isProduction(): bool
```

Comment:
so what do you thinks about some one want to use this package in project without env file config ?

Reply:
Think of it this way, this project is dependent on .env, you just didn't know it yet.

That is the only way to keep private credentials away from Git.

Unless your not concerned about security, in which case a professional project won't use your library.

Reply:
But I think I understand what your saying, but the problem I have with your code was setting constants for dev environment.

I didn't think it was appropriate. Setting environment should not be part of the project responsibility.

Reply:
The library won't need .env if you remove the environment check from it.

Reply:
I think it will be better to remove the .env and see how to use the API of prod or dev according to the config.

In fact if it was a specific package to a Laravel or Symfony project, it will make more sense with the configuration in an env file.

Reply:
Since you've already suggested we move the url selection out of the code, .env is no longer necessary.